src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -191,23 +194,20 @@
   }
 }
 
-// TODO: improve how this is styled. Styling just for the DQL editor.
-.osdQueryEditor__body .globalFilterGroup__wrapper {
-  margin-top: -$euiSizeS;
-}
-
```

Comment:
Why did we delete this?

Reply:
Good catch, I have reverted the change back.